packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -201,6 +202,14 @@ function Navbar() {
     [setLightTheme, setDarkTheme],
   );
 
+  const windowSize = useWindowSize();
+
+  useEffect(() => {
+    if (windowSize === windowSizes.desktop) {
+      setSidebarShown(false);
+    }
+  }, [windowSize, sidebarShown]);
```

Comment:
now here sidebarShown is not used in the effect anymore so it's not necessary to add it to the deps :)

Reply:
Indeed ! I made the changes